Arelle/Arelle
# Load OIM taxonomy plugin

#### Reason for change
Allow loading from json OIM taxonomy files (being drafted)

#### Description of change
Checks if the json file is an OIM taxonomy file and if so loads it into a ModelDocument

#### Steps to Test
Compare loaded taxonomy from OIM vs xsd file.
Check for different errors loading from OIM than xsd.
Check loadability of xsd with cmd line arg --saveOIMschemafile

**review**:
@Arelle/arelle
